src/_posts/languages/ruby/rails/2000-01-01-start.md

```diff
@@ -73,6 +73,12 @@ Then set the following environment variable:
 * `DATABASE_URL`
   → Copy the value of `SCALINGO_MYSQL_URL` and replace 'mysql://' by 'mysql2://'
 
+Can be automated in your `config/database.yml`:
```

Comment:
```suggestion
* `DATABASE_URL`: Copy the value of `SCALINGO_MYSQL_URL` and replace 'mysql://' by 'mysql2://'\\
  This can be automated in `config/database.yml`:
  ```erb
  url: <%= ENV["DATABASE_URL"]&.gsub(/mysql:/, 'mysql2') %>
  ```
```